status-go-version.json

```diff
@@ -1,9 +1,9 @@
 {
     "_comment": "THIS SHOULD NOT BE EDITED BY HAND.",
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
-    "owner": "status-im",
+    "owner": "qfrank",
     "repo": "status-go",
-    "version": "v0.94.1",
-    "commit-sha1": "1c549f7d714b489473626b2e933c703c2d73299d",
-    "src-sha256": "1w3rl1n3bb7a4qpa3m27ksqpm2avhdfwchbynphbrnj3rshnfzc5"
+    "version": "sync_clear_history",
```

Comment:
No, this is wrong. You cannot use a fork in `status-go-version.json`, nor should you use a branch.

It should be a tagged release.

Reply:
its just temporary for testing

Reply:
Well, that's the only thing I saw wrong, since I can't review the clojure changes.